lib/src/domain/entity/cast.dart

```diff
@@ -1,9 +1,24 @@
+import 'package:floor/floor.dart';
+
 import '../../core/util/strings.dart';
+import 'movie.dart';
+
+@Entity(
+  tableName: 'MovieCast',
```

Comment:
Maybe you can move the tableName to a const

Reply:
Done